pkg/resourceinterpreter/configurableinterpreter/configmanager/accessor.go

```diff
@@ -0,0 +1,97 @@
+package configmanager
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	configv1alpha1 "github.com/karmada-io/karmada/pkg/apis/config/v1alpha1"
+)
+
+// CustomConfiguration provides base information about custom interpreter configuration
+type CustomConfiguration interface {
+	GetConfigurationName() string
+	GetConfigurationTargetGVK() schema.GroupVersionKind
```

Comment:
```suggestion
	Name() string
	TargetResource() schema.GroupVersionKind
```
method names are redundance.